js/npm/restify/Error.hx

```diff
@@ -0,0 +1,431 @@
+package js.npm.restify;
+
+typedef ErrorOptions = {
+    ?message : Dynamic,
+    ?statusCode : Int
+}
+
+@:native('errors.HttpError')
+extern class HttpError extends js.Error implements js.extern.Extern<'restify'>
```

Comment:
I thought it was a standard meta linking to a JS name/class. But finally it looks like it was specific to the extern system I port it from. Will remove too